crates/solver/src/main.rs

```diff
@@ -216,8 +216,13 @@ struct Arguments {
     eden_api_url: Url,
 
     /// The API endpoint of the Flashbots network for transaction submission.
```

Comment:
Maybe add comment here for why one would specify multiple urls.

Reply:
Indeed. Added.